### DISCUSSION TITLE
Plans for a PHP SDK client?

### DISCUSSION THREAD

**Discussion by benoit-waldmann** [openfga/community#327] [Ideas]
Hello,

The lack of a PHP SDK client for this project limits integration options for PHP-based applications.

I'm currently working with your project and find it very useful. I was wondering if there are any plans to develop a PHP SDK client in the near future.

A PHP SDK would greatly benefit developers who primarily work with PHP-based applications, making it easier to integrate your project into their existing systems.

If there are no current plans for a PHP SDK, would you be open to community contributions in this area? I'd be interested in knowing your thoughts on this and any potential roadmap for SDK development.

Thank you for your time and consideration.

Best regards,
Ben

**Comment by rhamzeh** [comment]
Hi @benoit-waldmann - we do have a PHP SDK planned in the roadmap https://github.com/orgs/openfga/projects/1?pane=issue&itemId=49602502

However it's not planned soon as some other tasks are scheduled first esp regarding improving our testing scenarios across SDKs. As that work nears completion, a huge hurdle to unvlock new SDKs will be crossed

**Reply by benoit-waldmann** [reply]
Thank you very much for your answer. We are very excited!

**Reply by dyeam0** [reply]
Hi @benoit-waldmann, a little delayed in letting you know about this, but heads up, we have an experimental PHP SDK!
See https://github.com/openfga/roadmap/issues/40#issuecomment-2556186225

Note though, it's still experimental. It's not an officially maintained SDK (yet) but was created by none other than our own @evansims! You can give it a try and hope it helps you out.
